spec/features/user_logs_in_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'rails_helper'
```

Comment:
Your `rails_helper` already includes your `spec_helper`, no need to require that for the second time here, nor in any other specs